scribejava-apis/src/main/java/com/github/scribejava/apis/FlickrApi.java

```diff
@@ -34,7 +44,13 @@ public String getAccessTokenEndpoint() {
      */
     @Override
     public String getAuthorizationUrl(OAuth1RequestToken requestToken) {
-        return "https://www.flickr.com/services/oauth/authorize?oauth_token=" + requestToken.getToken();
+        String authUrl = String.format(AUTHORIZE_URL, requestToken.getToken());
+
+        if (perm != null) {
+            authUrl += "&perms=" + perm;
+        }
+
+        return authUrl;
```

Comment:
Isn't it the 'scope' in terms of OAuth? just another GET-param name.
It's better to add method getAuthorizationUrl with OAuthConfig as argument in DefaultApi10a (as it is in DefaultApi20).
And override it here, in FlickrApi.

Will it solve your issue?